linux/LinuxProcessList.c

```diff
@@ -763,6 +763,42 @@ static bool LinuxProcessList_readStatmFile(LinuxProcess* process, openat_arg_t p
    return r == 7;
 }
 
+static bool LinuxProcessList_checkPidNamespace(Process *process, openat_arg_t procFd) {
+   FILE *statusfile = fopenat(procFd, "status", "r");
+   if (!statusfile)
+      return false;
+
+   while (true) {
+      char buffer[PROC_LINE_LENGTH + 1];
+      if (fgets(buffer, sizeof(buffer), statusfile) == NULL)
+         break;
+
+      if (!String_startsWith(buffer, "NSpid:"))
+         continue;
+
+      char *ptr = buffer;
+      int pid_ns_count = 1;
+      while(!iswdigit(*ptr))
```

Comment:
Why would `/proc/<pid>/status` contain non-ascii characters?
Htop generally does not support reading non-ascii values from the kernel.

Reply:
You are right. `/proc/<pid>/status` won't contain non-ascii characters. I think `isdigit` should be fine here. `iswdigit` is an overkill.